base/docs/helpdb/Base.jl

```diff
@@ -914,9 +914,9 @@ generate an array of such random numbers.
 randexp
 
 """
-    chop(string)
+    chop(string, i = 1)
```

Comment:
I can change if you really want but I like the consistency of using s for string and i for integer.